python/pyspark/sql/connect/dataframe.py

```diff
@@ -633,6 +633,48 @@ def explain(self) -> str:
         else:
             return ""
 
+    def createGlobalTempView(self, name: str) -> None:
+        """Creates a global temporary view with this :class:`DataFrame`.
+
+        The lifetime of this temporary view is tied to this Spark application.
+
+        .. versionadded:: 3.4.0
+
+        Parameters
+        ----------
+        name : str
+            Name of the view.
+
+        Returns
+        -------
+        None
```

Comment:
```suggestion

        Returns
        -------
        None
```

I guess we won't need this for `None`.